packages/core/src/lib/relay/index.ts

```diff
@@ -96,40 +98,63 @@ class Relay extends GossipSub implements IRelay {
       return { recipients: [] };
     }
 
-    return this.publish(this.pubSubTopic, msg);
+    return this.gossipSub.publish(this.pubSubTopic, msg);
   }
 
   /**
    * Add an observer and associated Decoder to process incoming messages on a given content topic.
    *
    * @returns Function to delete the observer
    */
-  addObserver<T extends IDecodedMessage>(
-    decoder: IDecoder<T>,
+  public subscribe<T extends IDecodedMessage>(
+    decoders: IDecoder<T>[],
```

Comment:
```suggestion
    decoders: IDecoder<T>[] | IDecoder<T>,
```

Could make the API slight easier to use.

Reply:
Added